sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -1323,6 +1322,34 @@ async def test_async_queue_receiver_alive_after_timeout(self, servicebus_namespa
                     messages = await receiver.receive_messages()
                     assert not messages
 
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', dead_lettering_on_message_expiration=True, lock_duration='PT5M')
+    async def test_queue_receive_keep_conn_alive_async(self, servicebus_namespace_connection_string, servicebus_queue, **kwargs):
+        async with ServiceBusClient.from_connection_string(
+                servicebus_namespace_connection_string, logging_enable=False) as sb_client:
+
+            sender = sb_client.get_queue_sender(servicebus_queue.name)
+            receiver = sb_client.get_queue_receiver(servicebus_queue.name)
+
+            async with sender, receiver:
+                await sender.send_messages([Message("message1"), Message("message2")])
+
+                messages = await receiver.receive_messages(max_batch_size=20, max_wait_time=5)
+                receiver_handler = receiver._handler
+                assert len(messages) == 2
+                await asyncio.sleep(4 * 60 + 5)  # 240s is the service defined connection idle timeout
```

Comment:
I'm very torn about this; because having a 4 minute long livetest (8 with async) is less-than-desirable, but it's a very good test to have.  I see a few options.
1. keep this as is, cut test-time elsewhere.
2. move this into stress tests (doesn't have to use the stress test framework, just to encapsulate "longer-running tests")
3. reexamine running tests in parallel with the engsys team. (This may be more of a "do this alongside as a hail-mary" sort of thing.)

LMK if you see any clever options that I don't.  We can (and probably should) create an issue to track the test perf improvements, our runtimes have gotten long in the tooth again.

Reply:
as per discussion OOB, we're leaning towards option 1 "keep this as is, cut test-time elsewhere"

To cut test-time else where, we could:

1. mgmt livetest use recording, but have some nightly scheduled test running using real resource
2. decrease lock duration time in some test case as "lock_duration" is configurable now

Reply:
created a issue to track the work: https://github.com/Azure/azure-sdk-for-python/issues/13056